Assets/Scripts/Combat/Abilities/FireBall.cs

```diff
@@ -28,7 +28,7 @@ private enum FireballCyclePhase { Normal, UnstableWarning, Unstable };
         private const float FireballHeight = 7f;
         private const float FireballScale = 0.08f;
         private float TargetFireballSize = 0.15f;
-        private const float FireballGrowth = 0.025f;
+        private const float FireballGrowth = 2f;
```

Comment:
Woah woah woah why

Reply:
Using delta time now

Reply:
![image](https://user-images.githubusercontent.com/24488852/101963630-29cc5400-3bdd-11eb-9bb8-35bc615e9605.png)
The size increase now is... even bigger